superset/migrations/versions/2024-05-01_10-52_58d051681a3b_add_catalog_perm_to_tables.py

```diff
@@ -45,10 +49,21 @@ def upgrade():
         "slices",
         sa.Column("catalog_perm", sa.String(length=1000), nullable=True),
     )
+
+    if not table_has_index(perm_table, perm_index_view_menu):
+        op.create_index(op.f(perm_index_view_menu), perm_table, ["view_menu_id"])
+
+    if not table_has_index(perm_table, perm_index_permission_id):
+        op.create_index(op.f(perm_index_permission_id), perm_table, ["permission_id"])
+
     upgrade_catalog_perms(engines={"postgresql"})
 
 
 def downgrade():
     downgrade_catalog_perms(engines={"postgresql"})
+    if table_has_index(perm_table, perm_index_permission_id):
+        op.drop_index(op.f(perm_index_permission_id), table_name=perm_table)
+    if table_has_index(perm_table, perm_index_view_menu):
+        op.drop_index(op.f(perm_index_view_menu), table_name=perm_table)
```

Comment:
@betodealmeida any thoughts whether you think this change should stay with the new migration or in a different PR? My thought is to keep them together so that tables don't get out of sync, but I also see the benefit of splitting them up.